server/mailet/rate-limiter/pom.xml

```diff
@@ -47,6 +47,7 @@
         <dependency>
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-core</artifactId>
+            <scope>provided</scope>
```

Comment:
Q: Why do you add this scope?

Reply:
Similar to other dependencies, for external jar

Reply:
As far as I read on stackoverflow and what I understand, this is to avoid duplicating the dependency when importing the JAR to another project (here is james-server-core dependency and james project).